PackageSync.sublime-settings

```diff
@@ -24,6 +24,9 @@
 		"*.sublime-package"
 	],
 
+	// Toggle to determine whether to synchronize user setting for this package (`PackageSync.sublime-settings`).
+	"sync_package_sync_settings": false,
```

Comment:
Please add the following to the comment as well
`Caution: Use this feature very cautiously. This overrides your PackageSync settings across the synced devices. If you wish to keep different sync folder paths across the devices, DO NOT set this as True`


Also, please add the configuration and the corresponding comments to the README.md file.